### PR TITLE
Van task

### PR DESCRIPTION
**Pull request by kpvarma**

### CODE REVIEW

File: app/models/client_user.rb [hunk 1]
```diff
@@ -15,6 +15,10 @@ def display_name
     [first_name, last_name].compact.join(" ")
   end
   
+  def display_initials
```
Comment (kpvarma) [inline app/models/client_user.rb]: what if last name is nil?
@vanirankar

File: app/models/concerns/itslabel/scopes/client_user_scopes.rb [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+
+  included do
+
+    # Scopes
+    scope :search, lambda {|query| where("
+      LOWER(client_users.first_name) LIKE LOWER('%#{query}%') OR\
+      LOWER(client_users.last_name) LIKE LOWER('%#{query}%') OR\
+      LOWER(client_users.mobile_number) LIKE LOWER('%#{query}%') OR\
+      LOWER(client_users.email) LIKE LOWER('%#{query}%') OR\
+      LOWER(client_users.country) LIKE ('%#{query}%')
```
Comment (kpvarma) [inline app/models/concerns/itslabel/scopes/client_user_scopes.rb]: LOWER(query) for country missing

File: app/models/concerns/itslabel/scopes/translation_scopes.rb [hunk 3]
```diff
@@ -7,15 +7,18 @@ module Itslabel::Scopes::TranslationScopes
...
       LOWER(translations.input_language) LIKE LOWER('%#{query}%') OR\
 
       LOWER(translations.output_phrase) LIKE LOWER('%#{query}%') OR\
-      LOWER(translations.output_language) LIKE LOWER('%#{query}%') OR\
-      LOWER(translations.output_description) LIKE LOWER('%#{query}%') OR\
+      LOWER(translations.output_language) LIKE LOWER('%#{query}%') 
+    
+
+    ")}
+     scope :advsearch, lambda {|query| where("
```
Comment (kpvarma) [inline app/models/concerns/itslabel/scopes/translation_scopes.rb]: do we really need advsearch scope. if not please remove.

File: app/models/translation.rb [hunk 4]
```diff
@@ -24,8 +24,8 @@ class Translation < ApplicationRecord
   validates :output_phrase, presence: true, length: {maximum: 256}, allow_blank: false
   validates :output_language, presence: true, :inclusion => {:in => LANGUAGES, :message => "is not a valid language" }
 
-  validates :input_description, length: {maximum: 1024}, allow_blank: true
-  validates :output_description, length: {maximum: 1024}, allow_blank: true
+  #validates :input_description, length: {maximum: 1024}, allow_blank: true
```
Comment (kpvarma) [inline app/models/translation.rb]: please remove

**Comment by vanirankar** [comment]
Pls check for profile

File: app/controllers/admin/translations_controller.rb [hunk 5]
```diff
@@ -83,12 +83,45 @@ def destroy
 
     private
 
+        def apply_filters
```
Comment (kpvarma) [inline app/controllers/admin/translations_controller.rb]: We need to optimize this. however if this works, we are good for now.

**Pull request merged**